shepherd/Utils.py

```diff
@@ -139,3 +139,54 @@ class STATE():
     WAIT = "wait"
     TELEOP = "teleop"
     END = "end"
+
+class INGREDIENT():
+    BREAD = "Bread"
+    VEGETABLES = "Vegetables"
+    MEAT = "Meat"
+    CHEESE = "Cheese"
+    # COOKED_MEAT = "COOKED " + MEAT
+    # COOKED_BREAD = "COOKED " + BREAD
+    # COOKED_CHEESE = "COOKED " + CHEESE
+
+class RECIPIES():
+    BREADSTICKS = "Breadsticks"
+    SALAD = "Salad"
+    COLD_CUTS = "Cold Cuts"
+    TURKEY_SANDWICH = "Turkey Sandwich"
+    CAESAR_SALAD = "Caesar Salad"
+    MEAT_AND_POTATOES = "Meat and Potatoes"
+    HAMBURGER = "Hamburgers"
+    GRILLED_CHEESE = "Grilled Cheese"
+    JALAPENO_POPPERZ = "Jalapeno Popperz"
+    CHEESEBURGER = "Cheeseburger"
+    SHEPHERDS_PIE = "Sherpherd's Pie"
+    CHEESE_AND_CRACKERS = "Cheese and Crackers"
+
+class RECIPE_PROBABILITIES():
+    Ingredients = "Ingredients"
+    Cooked = "Cooked"
+    RECIPE_PROBABILITIES = {0: [], 1: [], 2:[], 3:[]}
```

Comment:
This seems like the same variable as the "mappings" variable in teleop_recipe_map. Perhaps that class should be removed and this variable be used instead